api/CacheStorage.json

```
@@ -6,17 +6,11 @@
         "support": {
           "chrome": {
             "version_added": "40",
-            "notes": [
-              "Accessible from <code>Window</code> from version 43.",
-              "Accessible from <code>WorkerGlobalScope</code> from version 43."
-            ]
+            "notes": "Version 40 to 42: supported in service workers only. Version 43 enabled use in main thread and other worker types."
```

Since this happened five years ago and isn't getting any younger, let's put this in terms that shows it's historical and let `worker_support` do most of the, um, work. 😃 

```suggestion
            "notes": "Before version 43, this is supported in service workers only."
```

I've put a suggestion here for Chrome and Chrome for Android, but could we also:

- `opera_android`: add the same note, except with version 30
- `opera_android`: add the same note, except with version 30
- `samsunginternet_android`: remove all the notes (not applicable)
- `webview_android`: Apply the same change